service/ResponsibilityService.ts

```diff
@@ -0,0 +1,22 @@
+import * as dotenv from 'dotenv';
+dotenv.config();
+import { Responsibility } from '../model/Responsibility';
+import axios from 'axios';
+axios.defaults.baseURL = process.env.API_URL;
+
+
+export default class ResponsibilityService {
+
+  URL: string = '/job-spec'; 
+
+  async getResponsibilitiesForJob(): Promise<Responsibility[]>{
```

Comment:
this class isn't being called, think you can delete

Reply:
Class and corresponding unit tests deleted